unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInputWrapper.cs

```diff
@@ -0,0 +1,90 @@
+using Builder;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class BuilderInputWrapper : MonoBehaviour
```

Comment:
This is a wrapper for DCLBuilderInput class, set that functionality on the InputController will add lots of code to the InputController that most likely won't be used most of the time. Since this is a different approach to the input I think that it is better to have its own class

Reply:
I think we can ultimately use said `InputController` but if the builder is meddling in between let's leave if for another PR.

Reply:
Added to the document for future revision